### PR TITLE
Update nats docs

### PR DESCRIPTION
**Pull request by aashikam**
## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/nats-basic-client/publisher.bal [hunk 1]
```diff
@@ -12,7 +12,7 @@ public function main() {
     // Initializes a producer.
     nats:Connection connection = new("nats://localhost:4222");
     nats:Producer producer = new(connection);
-    while (message != ESCAPE) {
+    while (true) {
```
Comment (daneshk) [inline examples/nats-basic-client/publisher.bal]: why are we change the condition to true here

Comment (aashikam) [reply examples/nats-basic-client/publisher.bal]: As this condition is checked again inside the `if` condition, we decided to remove this.

Comment (daneshk) [reply examples/nats-basic-client/publisher.bal]: shall we remove the if condition?. yes, It doesn't make sense to have if condition.

Comment (aashikam) [reply examples/nats-basic-client/publisher.bal]: Resolved

**Pull request merged**